src/pages/steps/AdditionalInformationStep/OfferScore.tsx

```diff
@@ -154,10 +154,25 @@ const OfferScore = ({ completedFields, offerId, scope, ...props }: Props) => {
   // @ts-expect-error
   const offer: Offer | undefined = getOfferByIdQuery.data;
 
-  const hasTheme: boolean = offer?.terms.some(
-    (term) => term.domain === 'theme',
+  // For these types there are currently no themes available
+  const eventTypesWithNoThemes: string[] = [
+    EventTypes['Eet- of drankfestijn'],
+    EventTypes['Festiviteit'],
+    EventTypes['Fiets- of wandelroute'],
+    EventTypes['Party of fuif'],
+    EventTypes['Spel of quiz'],
+  ];
```

Comment:
Would maybe put this in `EventTypes` as well

Reply:
Yes good suggestion 👍